modules/porous_flow/test/tests/sinks/tests

```diff
@@ -160,6 +160,54 @@
     issues = '#7333'
     design = 'porous_flow/tests/sinks/sinks_tests.md porous_flow/boundaries.md porous_flow/doc/tests/sinks.tex'
   [../]
+  [./s11]
```

Comment:
Also need a test when you specify pressure, not fluid_phase.   I think it'd be nicer to use CSVDiff and a PorousFlowHeatEnergy Postprocessor that computes the total heat energy in the system, where you can show analytically that your heat energy is increasing according to flux_function (which you could set non-trivially to strengthen the test).